backend/src/main/java/ca/bc/gov/app/exception/NoClientDataFound.java

```diff
@@ -6,8 +6,12 @@
 
 @ResponseStatus(HttpStatus.NOT_FOUND)
 public class NoClientDataFound extends ResponseStatusException {
+
+  private static final long serialVersionUID = -4060998577310972210L;
```

Comment:
You don't need to add the serial version anymore. This is only used for binary serialization and serves no purpose when handling JSON data

Reply:
It can still be a good practice in case the base class's serialVersionUID changes in future versions. It ensures that the custom exception class has a consistent serialVersionUID across different versions of the code, even if the base class changes.